modules/okenetwork/variables.tf

```diff
@@ -35,8 +35,9 @@ variable "lb_subnet_type" {
   type = string
 }
 
+variable "public_lb_ports" {}

```

Comment:
please set type list(string)

Reply:
Have set it to below to match the variable definition in variables.tf

variable "public_lb_ports" {
  type = list(number)
}